common/models/role-mapping.js

```diff
@@ -17,6 +17,15 @@ module.exports = function(RoleMapping) {
   RoleMapping.APP = RoleMapping.APPLICATION = 'APP';
   RoleMapping.ROLE = 'ROLE';
 
+  RoleMapping.getRelatedModels = function() {
+    if (!this.userModel) {
```

Comment:
`setup` is still too early to resolve the related models as they are guaranteed to be available in the registry when `setup` is called.

I'll rename the method to `resolveRelatedModels`.